go.mod

```
@@ -1,81 +1,87 @@
 module github.com/tendermint/tendermint
 
-go 1.21
+go 1.21.13
```

```suggestion
go 1.21
```

still think this should be minimal